stackslib/src/net/p2p.rs

```diff
@@ -3281,8 +3309,8 @@ impl PeerNetwork {
         });
 
         for broken_neighbor in broken_p2p_peers.into_iter() {
-            debug!(
-                "{:?}: De-register dead/broken neighbor {:?}",
+            info!(
+                "{:?}: De-register and ban dead/broken neighbor {:?} from epoch2x block download",
                 &self.local_peer, &broken_neighbor
```

Comment:
How often does this info log show up?

Reply:
I haven't seen it yet after 18.6 hours of operation on mainnet.  That's not surprising though since this code path is only ever exercised during IBD.